src/Entity/LtiInstance.php

```diff
@@ -15,14 +15,14 @@
  *  along with this program; if not, write to the Free Software
  *  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  *
- *  Copyright (c) 2019 (original work) Open Assessment Technologies S.A.
+ *  Copyright (c) 2020 (original work) Open Assessment Technologies S.A.
```

Comment:
Just a stupid question, since class is "old" (it's just a renaming) shouldn't it be `2019-2020` instead of just `2020`?

Reply:
Github sees it as a file change but actually it's a new entity, new file, so I think it makes sense